cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -1591,4 +1591,4 @@ def _write_gcp_snap_config(component):
     if gcp_creds_env_key not in daemon_env:
         daemon_env += '{}={}\n'.format(gcp_creds_env_key, creds_path)
         daemon_env_path.parent.mkdir(parents=True, exist_ok=True)
-        daemon_env_path.write_text(daemon_env)
+        daemon_env_path.write_text(daemon_env)
```

Comment:
I'd let this slide, but our automation runs flake8 when building the charms, and the newline change here will cause it to fail:

```
$ flake8 reactive/
reactive/kubernetes_master.py:1594:47: W292 no newline at end of file
```